app/domain/errors.rb

```diff
@@ -25,6 +25,11 @@ module Conjur
       code: "CONJ00065E"
     )
 
+    InvalidProxy = ::Util::TrackableErrorClass.new(
+      msg:  "Invalid proxy '{0-proxy-ip-address}'",
```

Comment:
@shulifink can you review please?

Reply:
Fine

Reply:
Is this the same as the trusted proxy list? If so, do we want "Invalid trusted proxy '{0-proxy-ip-address}'"